src/inventory/weapons/TurretGun.h

```diff
@@ -0,0 +1,45 @@
+/*
+    Created by Mark Chen 3/30/2017
+*/
+#ifndef TURRETGUN_H
+#define TURRETGUN_H
+#include <string>
+
+#include "InstantWeapon.h"
+#include "../../audio/AudioManager.h"
+#include "../../sprites/Renderer.h"
+#include "../../sprites/SpriteTypes.h"
+
+
+using std::string;
```

Comment:
while this is valid and should be ok, avoid all usings in headers. it is kinder to people including the files to not have namespace lookups they dont want.